src/modules/users/shemas/cvs.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from "zod";
+import i18n from "@/i18n";
+
+export const getCvFormSchema = () =>
+    z.object({
+        name: z.string().min(1, { message: i18n.t("fieldErrors.nameRequired") }),
+        description: z.string().min(1, { message: i18n.t("fieldErrors.descriptionRequired") }),
+        education: z.string().min(1, { message: i18n.t("fieldErrors.educationRequired") }),
+    });
```

Comment:
you can reuse the schema I used for cv details component

import { CvDetailsSchemaType, getCvDetailsSchema } from "@/modules/cvs/components/details/schemas";